repository_service_tuf/helpers/cli.py

```diff
@@ -75,10 +74,9 @@ def create_artifact_payload_from_filepath(
 ) -> Dict[str, Any]:
     """
     Create the payload for the API request of `POST api/v1/artifacts/`.
-
     The blake2b-256 cryptographic hash function is used to hash the file.
 
-    :param filepath: path to the artifact to be added to the metadata.
+    :param filepath: The file path to calculate the hash.
```

Comment:
For future: please make such documentation changes in separate commits and document that this is required from `make docs`. I am asking this as those changes are not related to the main changes in any way.
I had the same problem before.